README.md

```diff
@@ -145,17 +145,17 @@ a == null ? undefined : delete a.b      // this is what we get, really
 
 ## Not supported
 
-The following are not supported for lack of real-world use cases:
+The following are not supported due to lack of real-world use cases:
```

Comment:
This is also correct as-is; this change needs an indefinite article a la “due to a lack”

Reply:
You're right. "[D]ue to a lack of real-world use cases:" it is.